pkgs/top-level/aliases.nix

```
@@ -573,6 +573,7 @@ mapAliases ({
   transmission_gtk = transmission-gtk; # added 2018-01-06
   transmission_remote_gtk = transmission-remote-gtk; # added 2018-01-06
   transporter = throw "transporter has been removed. It was archived upstream, so it's considered abandoned.";
+  trilium = throw "trilium has been removed. Please use trilium-desktop instead.";
```

```suggestion
  trilium = throw "trilium has been removed. Please use trilium-desktop instead."; # added 2020-04-29
```

I know the attribute just above doesn't do this, but it would be nice to be relatively uniform with the rest of  the file.